README.md

```diff
@@ -117,9 +117,39 @@ AMIs can be automatically uploaded to AWS by specifying the following flags:
 | --aws-bucket   | Target S3 bucket name for intermediate storage when creating AMI |
 | --aws-region   | Target region for AWS uploads                                    |
 
-*Note: These flags must all be specified together. If none are specified, the AMI is exported to the output directory.*
+*Notes:*
+- *These flags must all be specified together. If none are specified, the AMI is exported to the output directory.*
+- *The bucket must already exist in the selected region, bootc-image-builder will not create it if it is missing.*
+- *The output volume is not needed in this case. The image is uploaded to AWS and not exported.*
+
```

Comment:
Is it worth adding a tiny summary of the auth methods here? Something like:

The AMI uploader supports two authentication methods:

1) AWS credentials file
2) AWS credentials via environment

With hyper-links? Or is that too much?

Reply:
I just left those comments as the majority of them were there - you are right keeping it short does make sense here